sdcm/sct_events/health.py

```diff
@@ -23,11 +23,13 @@ class ClusterHealthValidatorEventClusterHealthCheck(SctEventProtocol, Protocol):
     done: Type[SctEventProtocol]
 
 
-class ClusterHealthValidatorEvent(ValidatorEvent, abstract=True):
-    NodeStatus: Type[SeverityLevelProtocol]
-    NodePeersNulls: Type[SeverityLevelProtocol]
-    NodeSchemaVersion: Type[SeverityLevelProtocol]
-    NodesNemesis: Type[SeverityLevelProtocol]
+class ClusterHealthValidatorEvent(SctEvent, abstract=True):
+    NodeStatus: Type[SctEventProtocol]
+    NodePeersNulls: Type[SctEventProtocol]
+    NodeSchemaVersion: Type[SctEventProtocol]
+    NodesNemesis: Type[SctEventProtocol]
+    Done: Type[SctEventProtocol]
+    Info: Type[SctEventProtocol]
 
     ClusterHealthCheck: Type[ClusterHealthValidatorEventClusterHealthCheck]
```

Comment:
I think you should need to remove this subevent (and Protocol used here.)

Reply:
removed